packages/data-context/src/sources/FileDataSource.ts

```diff
@@ -31,7 +31,7 @@ export class FileDataSource {
   }
 
   async getFilesByGlob (cwd: string, glob: string | string[], globOptions?: GlobbyOptions) {
-    const globs = Array.isArray(glob) ? glob : [glob]
+    const globs = ([] as string[]).concat(glob)
 
     const ignoreGlob = globOptions && Array.isArray(globOptions?.ignore) ? globOptions.ignore.concat('**/node_modules/**') : ['**/node_modules/**']
```

Comment:
Can we do something similar here? `globOptions.ignore` is typed as an array so we can drop the `Array.isArray` and do `const ignoreGlob = (globOptions?.ignore || []).concat('**/node_modules/**')`

Reply:
```suggestion
    const ignoreGlob = (globOptions?.ignore ?? []).concat('**/node_modules/**')
```